### PR TITLE
Sequence tagging demo

### PR DESCRIPTION
**Pull request by emailweixu**
Demo for CONLL 2000 shared task: http://www.clips.uantwerpen.be/conll2000/chunking/

### CODE REVIEW

File: demo/sequence_tagging/linear_crf.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+)
+
+chunk_evaluator(
+    name="chunk_f1",
+    input =[crf_decoding, chunk],
+    chunk_scheme="IOB",
+    num_chunk_types=11,
+)
+
+inputs(word, pos, chunk, features)
```
Comment (qingqing01) [inline demo/sequence_tagging/linear_crf.py]: word is not used in config? 
There is no need to use  `inputs` , it should be `inputs([word, pos, chunk, features])`, if used.

Comment (emailweixu) [inline demo/sequence_tagging/linear_crf.py]: inputs() accepts *args.
word is indeed not used for the reason of sharing the same data provider as rnn_crf.py

File: demo/sequence_tagging/dataprovider.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+    def get_features(pos):
+        if pos < 0:
+            return ['#B%s' % -pos] * num_features
+        if pos >= length:
+            return ['#E%s' % (pos - length + 1)] * num_features
+        return sequence[pos]
+
+    for i in xrange(length):
+        for pattern in patterns:
+            fname = '/'.join([get_features(i+pos)[f] for pos, f in pattern])
```
Comment (qingqing01) [inline demo/sequence_tagging/dataprovider.py]: frame?

**Comment by emailweixu** [comment]
Resolved merge conflict

**Pull request merged**